### DISCUSSION TITLE
RAG: Prácticas recomendadas de Azure AI Search: Spanish Session Resources

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#32] [Announcements]
This is the resources and discussion thread for this live stream: https://aka.ms/raghack/ES2
It will be updated with links after the session.

Recording: https://www.youtube.com/watch?v=JLf5kVgVFFA
Slides: https://github.com/microsoft/RAG_Hack/blob/main/slides/RAGHack-AzureAISearch-Spanish.pptx

Please ask any follow-up questions here, and the speakers will see them.

**Comment by fchopo** [comment]
Hello, 
The recording link is incorrect, as it redirects to the first session recording. Please, could you fix it?
Thanks!

**Reply by pamelafox** [reply]
Fixed, gracias!
